scripts/local-universe.py

```diff
@@ -45,9 +45,9 @@ def main():
     parser.add_argument(
         '--include',
         default='',
-        help='Command separated list of packages to include. If this option '
-        'is not specified then all packages are downloaded. E.g. '
-        '--include="marathon,chronos"')
+        help='Command (,) separated list of packages to include. For each '
```

Comment:
Yes. I think the previous behavior was a mistake. We should avoid suggesting that user should generate a universe that contains all of the packages.